src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,46 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-12#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .row
+      .col-md-4.order-md-2.order-sm-1.mb-4#side-links
+        - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+          %h4
+            Reviews
 
-  .comment_new.mt-3
-    = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+          .mb-4
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: accepted_reviews, as: :review,
+                     locals: { key: :accepted_reviews }
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: declined_reviews, as: :review,
+                     locals: { key: :declined_reviews }
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: open_reviews, as: :review,
+                     locals: { key: :open_reviews }
+
+          - open_reviews_for_staging_projects.each do |review|
+            %div
+              %i.fas.fa-info-circle.text-info
+              - staging_project = review.project
+              Is staged in
+              = link_to(review.by_project, staging_workflow_staging_project_path(staging_project.staging_workflow.project, staging_project.name))
+
+          - if bs_request.accept_at.present?
+            %div
+              %i.fas.fa-exclamation-circle.text-danger
+              - if BsRequest::FINAL_REQUEST_STATES.include?(bs_request.state)
+                Auto-accept was set to
+                %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
+                = succeed '.' do
+                  = l bs_request.accept_at, format: :only_date
+              - elsif bs_request.accept_at.past?
+                This request will be automatically accepted when it enters the 'new' state.
+              - else
+                This request will be automatically accepted in #{time_ago_in_words(bs_request.accept_at)}.
```

Comment:
Hm what's the difference to the `Auto-accept was set to` sentence above?

Reply:
Well, having a look at this... we think it's better to exclude that bunch of code from this PR. We still need to understand the reasoning behind the three conditions there.

This is the new card in the backlog: https://trello.com/c/8ogiLXqD/2677-introduce-messages-about-automatic-acceptance